CHANGES.md

```diff
@@ -4,7 +4,7 @@ Release Notes.
 
 8.10.0
 ------------------
-
+* Support Java thread pool metric collect.
```

Comment:
This PR only provides thread pool metrics of Undertow, but the change log makes me (and probably other users) feel that we support JDK's thread pool. WDYT @wu-sheng ?

Reply:
Updated, check the latest codes

Reply:
I noted that the V9.0.0 just display tomcat thread pool metric. Now I have many customized thread pool which created by "org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor". How can I collect the metric of my thread pool and display it in UI？Is need to write another customized  plugins or just need write some config?

Reply:
Check #146 you may be interested.

Reply:
I had checked #146 before i asked and i had read the code. The code does not  contains any METRIC about "CorePoolSize","MaximumPoolSize".
![image](https://user-images.githubusercontent.com/12442438/164586733-ae6f8a99-9058-4bd6-b952-4fbbaeaab95e.png)

I find a plugin more closer I wanted but hard coded[org.apache.skywalking.apm.plugin.undertow.worker.thread.pool.UndertowWorkerThreadPoolConstructorIntercept]. So I want to  konw if i have to write some customized plugin which the count is same to the ThreadPool count？Or have some config more convenient？

![image](https://user-images.githubusercontent.com/12442438/164589612-8157deb5-7356-4ccb-af93-bf48739a940d.png)

Reply:
Why only including two methods, there was a long discussion on that PR, you could check.

You definitely could write a new plugin to do whatever you want, but you should know, that the discussion context is recommended for you before you take action. There are risks to instrument all, which is why we don't do that.

Reply:
Thanks, i will check later

Reply:
@wu-sheng There are many thread pools in my project, such as ThreadPoolA and ThreadPoolB. If I want to monitor the metrics of these two thread pools separately, do I need to add a new plugin to the agent and two rules to the threadpool.ymal of OAP like:
\- name: ThreadPoolA
  exp: ThreadPoolA.avg(['metric_type', 'pool_name', 'instance', 'service'])
\- name: ThreadPoolB
  exp: ThreadPoolB.avg(['metric_type', 'pool_name', 'instance', 'service'])

Reply:
> Thanks, i will check later

Reply:
Learn MAL, your questions are irrelevant